src/server/http/max_payload_size.test.js

```diff
@@ -19,52 +19,34 @@
 
 import * as kbnTestServer from '../../test_utils/kbn_server';
 
-let kbnServer;
-async function makeServer({ maxPayloadBytesDefault, maxPayloadBytesRoute }) {
-  kbnServer = kbnTestServer.createServer({
-    server: { maxPayloadBytes: maxPayloadBytesDefault }
-  });
+let root;
+beforeAll(async () => {
+  root = kbnTestServer.createRoot({ server: { maxPayloadBytes: 100 } });
 
-  await kbnServer.ready();
+  await root.start();
 
-  kbnServer.server.route({
+  kbnTestServer.getKbnServer(root).server.route({
     path: '/payload_size_check/test/route',
     method: 'POST',
-    config: { payload: { maxBytes: maxPayloadBytesRoute } },
-    handler: function (req, reply) {
-      reply(null, req.payload.data.slice(0, 5));
-    }
+    config: { payload: { maxBytes: 200 } },
+    handler: (req, reply) => reply(null, req.payload.data.slice(0, 5)),
   });
-}
-
-async function makeRequest(opts) {
-  return await kbnTestServer.makeRequest(kbnServer, opts);
-}
+}, 30000);
```

Comment:
__note:__ I'm not sure if it's still considered as "normal" Jest unit test or I should put it into Jest `integration_tests`?

Reply:
Is there a reason you think this step might take 30 seconds? I'd say when you can't control the amount of time a test takes it should probably be in the integration suite.

Reply:
It's not stable on my laptop (as any other test that's running Kibana) - sometimes it may take 16s or so. Do you think we should move all these tests to `intergration_tests` or just reduce this timeout and see what happens on CI?

Reply:
I'd say move it to integration tests, just not sure why it would take 16 seconds to start a server...

Reply:
Okay, I'll move all the tests where I had to set this timeout to integration tests.